pytensor/xtensor/type.py

```diff
@@ -616,14 +720,10 @@ def expand_dims(
                 - int: the new size
                 - sequence: coordinates (length determines size)
```

Comment:
```suggestion

            - int: the new size
            - sequence: coordinates (length determines size)
```

no extra indentation and empty line between previous line and list should render as list instead of current [quoted list render](https://pytensor--1533.org.readthedocs.build/en/1533/library/xtensor/type.html#pytensor.xtensor.type.XTensorVariable.expand_dims)